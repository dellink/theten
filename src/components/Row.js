import React from 'react';
import { StyleSheet, View } from 'react-native';
import { CELL_MARGIN, CELL_SIZE } from '../config';

export default function Row() {
  return (
    <View style={styles.row}>
      <View style={styles.cell} />
      <View style={styles.cell} />
      <View style={styles.cell} />
      <View style={styles.cell} />
    </View>
  );
}

const styles = StyleSheet.create({
  row: {
    flexDirection: 'row',
  },
  cell: {
    backgroundColor: '#ddccbb',
    borderRadius: 5,
    height: CELL_SIZE,
    margin: CELL_MARGIN,
    width: CELL_SIZE,
  },
});
