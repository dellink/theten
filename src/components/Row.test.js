import React from 'react';
import { render } from '@testing-library/react-native';
import Row from './Row';

describe(Row.name, () => {
  it('should render correctly', () => {
    expect(render(<Row />).asJSON()).toMatchSnapshot();
  });
});
