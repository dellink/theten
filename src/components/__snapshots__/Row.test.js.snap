// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row should render correctly 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "#ddccbb",
          "borderRadius": 5,
          "height": 68,
          "margin": 5,
          "width": 68,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "#ddccbb",
          "borderRadius": 5,
          "height": 68,
          "margin": 5,
          "width": 68,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "#ddccbb",
          "borderRadius": 5,
          "height": 68,
          "margin": 5,
          "width": 68,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "#ddccbb",
          "borderRadius": 5,
          "height": 68,
          "margin": 5,
          "width": 68,
        }
      }
    />
  </View>
</View>
`;
